endpoints/openrtb2/auction_benchmark_test.go

```diff
@@ -88,6 +88,7 @@ func BenchmarkOpenrtbEndpoint(b *testing.B) {
 		func(ctx context.Context, id uint16) (vendorlist.VendorList, error) { return nil, nil },
 		currency.NewRateConverter(&http.Client{}, "", time.Duration(0)),
 		empty_fetcher.EmptyFetcher{},
+		nil,
```

Comment:
Consider using a NilSigner here instead to document what the argument is for.

Reply:
Yes, replaced in all occurrences